src/components/UI/Buttons/IconButton.vue

```diff
@@ -0,0 +1,141 @@
+<template>
```

Comment:
What is this template for? A general icon button? Maybe adding a comment near the top for each template will be helpful?